components/new-contribution-flow/StepProfile.js

```diff
@@ -1,43 +1,348 @@
+                    placeholder="160 Zion Ln.&#13;&#10;Temecula, CA&#13;&#10;90210"
+                    width="100%"
+                    height="150px"
+                    maxWidth={350}
+                    fontSize="LeadCaption"
+                    required
+                    onChange={e => onChange({ stepProfile: { ...data, [e.target.name]: e.target.value } })}
+                  />
+                )}
+              </StyledInputField>
+            </Box>
+            <Box width={1 / 2} mb={3} ml={1}>
+              <StyledInputField
+                name="country"
+                label={<FormattedMessage id="ExpenseForm.ChooseCountry" defaultMessage="Choose country" />}
+                htmlFor="country"
+              >
+                {inputProps => (
+                  <InputTypeCountry
+                    {...inputProps}
+                    locale={locale}
+                    onChange={value => {
+                      setLocale(value);
+                      onChange({ stepProfile: { ...data, country: value } });
+                    }}
+                    value={data?.country}
+                    required
+                  />
+                )}
+              </StyledInputField>
+            </Box>
+          </Flex>
+        )}
+        <P fontSize="SmallCaption" color="black.600">
+          <FormattedMessage
+            id="ContributionFlow.PublicContribution"
+            defaultMessage="Your name and contribution will be public."
+          />
+        </P>
+        <MessageBox type="info" fontSize="Caption" color="black.800" my={2} py={2}>
+          {renderInfoMessage(amount)}
+        </MessageBox>
+        <div>Newsletter</div>
+        <Flex width={1} alignItems="center" justifyContent="center">
+          <StyledHr width="100%" ml={3} borderColor="black.300" />
+        </Flex>
+        <Flex alignItems="center" mt={3}>
+          <P fontSize="Paragraph" mr={2}>
+            <FormattedMessage id="CreateProfile.AlreadyHaveAnAccount" defaultMessage="Already have an account?" />
+          </P>
+          <Link route={'/signin'} passHref>
+            <StyledLink fontSize="Paragraph">
+              <FormattedMessage id="signIn" defaultMessage="Sign In" />
+              &nbsp;→
+            </StyledLink>
+          </Link>
+        </Flex>
+      </Container>
+    );
+  };
+
+  // for logged in users donating once or recurrently, and new users donating recurrently
+  // TODO: bug with not selecting incognito if you change main profiles too often?
+  const renderHowDoYouWantToAppear = chosenProfile => {
+    const howDoYouWantToAppearOptions = [chosenProfile];
+    howDoYouWantToAppearOptions.push({
+      id: 'incognito',
+      type: 'USER',
+      isIncognito: true,
+      name: intl.formatMessage(messages['incognito']),
+    });
+
+    // const defaultValue = howDoYouWantToAppearOptions.find(option =>
+    //   option.id === data.id ? data.id : localProfile.id,
+    // );
+
```

Comment:
```suggestion
```

Reply:
Sorry, it's because it's still very WIP and was trying to figure stuff out. Comments will of course be gone in the final version!